src/components/ReportActionItem/TaskPreview.js

```diff
@@ -68,7 +70,10 @@ const TaskPreview = (props) => {
                     containerStyle={[styles.taskCheckbox]}
                     isChecked={isTaskCompleted}
                     onPress={() => {
```

Comment:
Coming from  [Unauthorized Task Reopening by Anonymous User in Incognito Mode- Inconsistency](https://github.com/Expensify/App/issues/23114)

We should have checked if an anonymous user is trying to mark a task as completed.